### PR TITLE
Remove deprecated test data generator code

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

`MakeTspPerformance` function was refactored in #1003, this PR replaces all deprecated function use with refactored function and adds support for adding all fields for a TSP Performance

## Setup
`make server_test`

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/160669841) for this change

### CODE REVIEW

File: pkg/models/transportation_service_provider_performance_test.go [hunk 1]
```diff
@@ -549,14 +692,14 @@ func (suite *ModelSuite) Test_FetchUnbandedTSPPerformanceGroups() {
...
-	suite.True(perfGroups[0].RateCycleEnd.Equal(foundPerfGroup.RateCycleEnd),
-		"RateCycleEnd in TSPP group did not match")
+	// suite.True(perfGroups[0].PerformancePeriodStart.Equal(foundPerfGroup.PerformancePeriodStart),
+	// 	"PerformancePeriodStart in TSPP group did not match")
+	// suite.True(perfGroups[0].PerformancePeriodEnd.Equal(foundPerfGroup.PerformancePeriodEnd),
+	// 	"PerformancePeriodEnd in TSPP group did not match")
+	// suite.True(perfGroups[0].RateCycleStart.Equal(foundPerfGroup.RateCycleStart),
+	// 	"RateCycleStart in TSPP group did not match")
+	// suite.True(perfGroups[0].RateCycleEnd.Equal(foundPerfGroup.RateCycleEnd),
+	// 	"RateCycleEnd in TSPP group did not match")
```
Comment (ralren) [inline pkg/models/transportation_service_provider_performance_test.go]: Are these suppose to be commented out?

Comment (Ronolibert) [reply pkg/models/transportation_service_provider_performance_test.go]: Oops! No, I'll uncomment that

File: pkg/awardqueue/awardqueue_test.go [hunk 2]
```diff
@@ -562,7 +637,7 @@ func (suite *AwardQueueSuite) Test_AssignTSPsToBands() {
 		t.Errorf("Failed to fetch TSPPerformances: %v", err)
 	}
 
-	expectedBands := []int{1, 1, 2, 3, 4}
+	expectedBands := []int{1, 1, 1, 1, 1}
```
Comment (mkrump) [inline pkg/awardqueue/awardqueue_test.go]: Is this still following the intent of the original test? `Test_AssignTSPsToBands` seems like it's about assigning TSPs to different quality bands, but wasn't sure if that was still relevant.

Comment (Ronolibert) [reply pkg/awardqueue/awardqueue_test.go]: Ahh, good catch. Previously the function just used hard coded quality bands but I changed it towards the end to specify the band. I'll fix this!

**Pull request merged**